common/views/components/EventScheduleItem/EventScheduleItem.js

```diff
@@ -1,83 +1,150 @@
-              </p>
-            );
-          })}
+    <li
+      className={`${spacing({ l: 0 }, { padding: ['left'] })} ${spacing(
+        { s: 4, l: 6 },
+        { padding: ['bottom'] }
+      )} ${spacing(
+        { s: 4 },
+        { margin: ['bottom'] }
+      )} border-color-smoke border-bottom-width-1`}
+    >
+      <div className="grid">
+        <div
+          className={`${grid({ s: 12, m: 12, l: 3, xl: 2 })} ${spacing(
+            { s: 2, l: 0 },
+            { margin: ['bottom'] }
+          )}`}
+        >
+          {event.times &&
+            event.times.map(t => {
+              const startTimeString = t.range.startDateTime.toString();
+              const endTimeString = t.range.endDateTime.toString();
+              return (
+                <p
+                  key={`${event.title} ${startTimeString}`}
+                  className={`${font({ s: 'HNM4' })} no-margin`}
+                >
+                  <time dateTime={startTimeString}>
+                    {formatTime(t.range.startDateTime)}
+                  </time>
+                  &mdash;
+                  <time dateTime={endTimeString}>
+                    {formatTime(t.range.endDateTime)}
+                  </time>
+                </p>
+              );
+            })}
         </div>
-        <div className={`${grid({s: 12, m: 12, l: 9, xl: 10})}`}>
+        <div className={`${grid({ s: 12, m: 12, l: 9, xl: 10 })}`}>
           <div>
-            {event.labels.length > 0 &&
-              <div className={spacing({s: 1}, {margin: ['bottom']})}>
+            {event.labels.length > 0 && (
+              <div className={spacing({ s: 1 }, { margin: ['bottom'] })}>
                 <LabelsList labels={event.labels} />
               </div>
-            }
-            <h3 className={`${font({s: 'WB6', l: 'WB5'})} ${spacing({s: 0}, {margin: ['top']})} ${spacing({s: 1}, {margin: ['bottom']})}`}>{event.title}</h3>
-            {event.place &&
-              <p className={`${spacing({s: 1}, {margin: ['bottom']})} ${font({s: 'HNL4'})}`}>{event.place.title}</p>
-            }
+            )}
+            <h3
+              className={`${font({ s: 'WB6', l: 'WB5' })} ${spacing(
+                { s: 0 },
+                { margin: ['top'] }
+              )} ${spacing({ s: 1 }, { margin: ['bottom'] })}`}
+            >
+              {event.title}
+            </h3>
+            {event.place && (
+              <p
+                className={`${spacing({ s: 1 }, { margin: ['bottom'] })} ${font(
+                  { s: 'HNL4' }
+                )}`}
+              >
+                {event.place.title}
+              </p>
+            )}
 
-            <p className={`${spacing({s: 2}, {margin: ['bottom']})} ${font({s: 'HNL5', m: 'HNL4'})}`} dangerouslySetInnerHTML={{__html: event.promoText}} />
+            <p
+              className={`${spacing({ s: 2 }, { margin: ['bottom'] })} ${font({
```

Comment:
This typeof stuff will need tidying up - but at least it looks really crap now, so we can do it as and when.